homeassistant/components/sensor/mqtt.py

```diff
@@ -28,6 +32,7 @@
 PLATFORM_SCHEMA = mqtt.MQTT_RO_PLATFORM_SCHEMA.extend({
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Optional(CONF_UNIT_OF_MEASUREMENT): cv.string,
+    vol.Optional(CONF_EXPIRE_AFTER, default=0): cv.positive_int,
```

Comment:
I can but i'd still have to check 0 and None because with 0 it would cause strange behaviour otherwise.